Backend/src/main/java/com/praksa/auction/WebSecurityConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.praksa.auction;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+public class WebSecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    @Override
```

Comment:
WebSecurityConfigurerAdapter is also deprecated now in new Spring Boot version. 
Please investigate filterChain method instead of using this configure method.
https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter
https://stackoverflow.com/questions/71348952/spring-security-without-the-websecurityconfigureradapter